frontend/public/module/k8s/index.ts

```diff
@@ -33,10 +33,12 @@ export type ObjectReference = {
 export type ObjectMetadata = {
   name?: string;
   generateName?: string;
+  uid?: string;
   annotations?: {[key: string]: string},
   namespace?: string,
   labels?: {[key: string]: string},
   ownerReferences?: OwnerReference[],
+  deletionTimestamp?: string | Date;
```

Comment:
Isn't this always a string?

Reply:
I am playing with type generation currently (please see https://github.com/openshift/console/pull/1731). The disscussion is still ongoing if to use a separate repo for the types, pulling them to our code base or not having them.

I found out that some types are generated as string (kubevirt types) but some are generated as Date (openshift types; https://github.com/openshift/origin/blob/master/api/swagger-spec/openshift-openapi-spec.json) 

So I figured it might not hurt to have the both options here (used by `@console/shared` selectors)

But I can remove this for now until the issue with types is resolved, if you prefer.